### DISCUSSION TITLE
Disentangle C-chain and X-chain from P-chain

### DISCUSSION THREAD

**Discussion by EricForgy** [avalanche-foundation/ACPs#59] [Brainstorming]
In my opinion, separating X-chain and C-chain from P-chain should happen before https://github.com/avalanche-foundation/ACPs/discussions/14.

As mentioned in this comment: https://github.com/avalanche-foundation/ACPs/discussions/14#discussioncomment-7437018

> If you don't mind me thinking out loud: if the P-chain becomes the "primenet" but still requires X and C to be validated in the same set, these two would, in my opinion, feel "bolted on" for legacy purposes rather than actually being relevant.

It seems to me that implementing #14 would be much cleaner and compelling if C-chain and X-chain were jetisoned as dependencies of P-chain.  

How difficult would that be? 

PS: Seeing suggestions like [this](https://twitter.com/kevinsekniqi/status/1737272060399796658) inspired me to bring this up.

PPS: I am currently listening to [this](https://www.youtube.com/watch?v=hJKWaba7IBk) and POG mentioned the posibility of having Subnet-only validator just verifying P-chain, but if P-chain were cleanly separated, then they could fully validate.

**Comment by Adventuremonkey** [comment]
I agree with Disentangling. Here is a more detailed proposal. https://github.com/avalanche-foundation/ACPs/discussions/88
